tests/rotate/expected/test1a.exit

```diff
@@ -0,0 +1 @@
+0
```

Comment:
@LeeBergstrand Do you think it makes sense to include all these expected output files in the repo, under `tests/rotate/expected`? The files are relatively small -- their total size comes to about 0.25 Mb. Thanks.